FunctionsSandboxLibrary/buildRequest.js

```diff
@@ -6,38 +6,8 @@ const encryptSecrets_1 = require("./encryptSecrets")
 const buildRequest = async (unvalidatedConfig) => {
   const config = (0, getRequestConfig_1.getRequestConfig)(unvalidatedConfig)
   const request = { source: config.source }
-  if (
```

Comment:
Just to confirm, this gets merged then the sandbox will no longer handle inline secrets right?
So it will be a breaking change to the `request` command trying to use inline secrets